### ISSUE TITLE
Account Backup 

### ISSUE THREAD

**Issue by realmenwearpink2** [b-lam/Resplash#160]
A simple account backup, that keeps track of the collections you have subscribed to for auto wallpaper would be great.
Save the trouble of re-subbing to all the collections when switching/wiping phones

**Comment by UnFunnyGuy** [comment]
@realmenwearpink2 are you talking about a cloud backup or local backup?

**Comment by realmenwearpink2** [comment]
> @realmenwearpink2 are you talking about a cloud backup or local backup?

A cloud backup.
Although I did just get a new phone, and doing the transfer with a cable brought my repainter account + all collections with it.
Not sure if it would be different had I done a cloud restore

**Comment by b-lam** [comment]
Yeah there isn't anything that I explicitly added to provide backup/restore functionality but I also just got a new phone and did a cable restore. To my surprise my account and collections got transferred over. I'm guessing the app data or something gets transferred over when you do that. I think local backup would be a bit easier to implement on my side if I was going to add something.
